lib/styles/components/_icon-button.scss

```diff
@@ -0,0 +1,232 @@
+@import '../settings/animations';
+@import '../settings/buttons';
+@import '../settings/icons';
+@import '../settings/media-queries';
+@import '../settings/spacings';
+@import '../settings/typography';
+
+@mixin setIconButtonAdditions($ripple: null, $border: null, $icon: null) {
+	@if $ripple != null {
+		.ripple {
+			background-color: $ripple !important;
+		}
+	}
+
+	@if $ripple == null {
+		.ripple {
+			display: none;
+		}
+	}
+
+	@if $border != null {
+		border: 1px solid $border;
+	}
+
+	@if $icon != null {
+		.a-iconButton {
+			&__icon,
+			&__loadingIcon {
+				color: $icon;
+			}
+		}
+	}
+}
+
+.a-iconButton {
```

Comment:
we extracted styles for `buttons` into a separate file only becasue Button is used outside of Vue components - in Blade files. I don't see any reason to follow this rule here. It's easier to read and maintain code when styles and HTML are in the same file

Reply:
Ok let it be, it's not so important now for me to keep it in separated file.